css/sekcje/header.css

```diff
@@ -0,0 +1,131 @@
+.header {
+  font-family: var(--fnt-primary);
+  font-style: normal;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  width: 1280px;
+  margin: 0 auto;
+  position: relative;
+  background-color: var(--clr-white);
+}
+
+.header__navigation {
+  display: flex;
+}
+
+.header__nav-list {
+  display: flex;
+  gap: 20px;
+  font-weight: 400;
```

Comment:
To nic nie robi bo w ul nie mamy tekstu (jest on jedynie w `a` a `a` nie bierze stylizacji z rodziców)